### PR TITLE
feat(coding-agent): show subagent model and effort in agents view

### PR DESCRIPTION
**Pull request by sethkarten**
## Summary

- show each subagent's model and effective reasoning effort directly in Agents View rows
- render stable information first: `name · model/effort · summary`, so variable summaries truncate before model details
- preserve labels for current and pre-`runtimeKind` daemons by reusing the existing subagent classifier
- omit disabled (`off`) effort while preserving real levels
- preserve confirmation prompts and narrow-width behavior

## Validation

- focused Agents View regression covers full and narrow ordering, empty summaries, disabled effort, scoped subagents, and legacy linkage fallback
- exact Biome check passed for all three changed files
- `git diff --check` passed
- independent static review approved exact head `8fdd8b3`

Earlier CI exposed missing theme initialization in the focused rendering test; `1432df0` fixed that failure and the empty-summary separator finding. `b587f98` addressed the disabled-effort finding. `8fdd8b3` addresses the pre-`runtimeKind` Macroscope finding. The focused test could not run locally because the available validation environment lacks `get-east-asian-width`; repository CI runs the complete dependency environment.

### CODE REVIEW

File: packages/coding-agent/src/modes/agents-view/agents-view-mode.ts [hunk 1]
```diff
@@ -2556,7 +2556,12 @@ export class AgentsViewMode implements Component, Focusable {
 		// "fix auth · Refactoring token validation". Hidden during delete/stop
 		// confirmations so the warning text stands alone.
 		const summaryText = !pendingDelete && !pendingKill ? row.summary.summary : undefined;
-		const titleContent = summaryText ? `${title} ${theme.fg("dim", `· ${summaryText}`)}` : title;
+		const modelLabel =
+			row.summary.runtimeKind === "subagent" && !pendingDelete && !pendingKill && row.summary.model
+				? `${row.summary.model.provider}/${row.summary.model.id}${row.summary.thinkingLevel ? `:${row.summary.thinkingLevel}` : ""}`
+				: undefined;
+		const suffixes = [summaryText, modelLabel].filter((suffix): suffix is string => suffix !== undefined);
```
[the remark this answers is not in the document]
Comment (sethkarten) [reply packages/coding-agent/src/modes/agents-view/agents-view-mode.ts]: Fixed in 1432df0: empty summary strings are filtered before composing suffix separators, and the focused render regression now covers the empty-summary case.

File: packages/coding-agent/src/modes/agents-view/agents-view-mode.ts [hunk 2]
```diff
@@ -2556,7 +2556,12 @@ export class AgentsViewMode implements Component, Focusable {
 		// "fix auth · Refactoring token validation". Hidden during delete/stop
 		// confirmations so the warning text stands alone.
 		const summaryText = !pendingDelete && !pendingKill ? row.summary.summary : undefined;
-		const titleContent = summaryText ? `${title} ${theme.fg("dim", `· ${summaryText}`)}` : title;
+		const modelLabel =
+			row.summary.runtimeKind === "subagent" && !pendingDelete && !pendingKill && row.summary.model
+				? `${row.summary.model.provider}/${row.summary.model.id}${row.summary.thinkingLevel ? `:${row.summary.thinkingLevel}` : ""}`
+				: undefined;
```
[the remark this answers is not in the document]
Comment (sethkarten) [reply packages/coding-agent/src/modes/agents-view/agents-view-mode.ts:2561]: Fixed in b587f98: explicit `off` effort is omitted while real effort levels remain visible; the focused render regression covers both cases.

File: packages/coding-agent/src/modes/agents-view/agents-view-mode.ts [hunk 3]
```diff
@@ -2552,11 +2552,16 @@ export class AgentsViewMode implements Component, Focusable {
...
 				? `${keyText("app.agents.delete")} again to ${row.section === "running" ? "stop" : "delete"}`
 				: styleRowTitle(row);
-		// Append the background summary as a dim suffix on the same line, e.g.
-		// "fix auth · Refactoring token validation". Hidden during delete/stop
-		// confirmations so the warning text stands alone.
+		// Keep stable model information ahead of the variable summary so narrow rows truncate the summary first.
 		const summaryText = !pendingDelete && !pendingKill ? row.summary.summary : undefined;
-		const titleContent = summaryText ? `${title} ${theme.fg("dim", `· ${summaryText}`)}` : title;
+		const modelLabel =
+			row.summary.runtimeKind === "subagent" && !pendingDelete && !pendingKill && row.summary.model
```
[the remark this answers is not in the document]
Comment (sethkarten) [reply packages/coding-agent/src/modes/agents-view/agents-view-mode.ts]: Fixed in 8fdd8b3: Agents View now reuses the existing runtimeKind-first legacy-linkage classifier, and the focused regression covers a pre-runtimeKind child identified by rlmChildId.

**Review by snimu — APPROVED** [review]
